draft-ietf-quic-transport.md

```diff
@@ -666,64 +666,226 @@ Version negotiation uses unprotected data.  The result of the negotiation MUST
+{{transport-parameter-definitions}}.  Use of the version fields from transport
+parameters is described in {{version-validation}}.
+
+
+### Transport Parameter Definitions
+
+An endpoint MUST include the following parameters in its encoded
+TransportParameters:
 
-The transport component of the handshake is responsible for exchanging and
-negotiating the following parameters for a QUIC connection.  Not all parameters
-are negotiated, some are parameters sent in just one direction.  These
-parameters and options are encoded and handed off to the crypto handshake
-protocol to be transmitted to the peer.
+SFCW (0x0000):
 
-#### Encoding
+: The initial stream level flow control parameter is encoded as an unsigned
+  32-bit integer.  The sender of this parameter indicates that the flow control
+  offset for all stream data sent toward it is this value.
 
-(TODO: Describe format with example)
+CFCW (0x0001):
 
-QUIC encodes the transport parameters and options as tag-value pairs, all as
-7-bit ASCII strings.  QUIC parameter tags are listed below.
+: The connection level flow control parameter contains the initial connection
+  flow control window encoded as an unsigned 32-bit integer.  The sender of this
+  parameter sets the byte offset for connection level flow control to this
+  value.  This is equivalent to sending a WINDOW_UPDATE ({{frame-window-update}}
+  for stream 0 immediately after completing the handshake.
 
-#### Required Transport Parameters {#required-transport-parameters}
+MSPC (0x0002):
 
-* SFCW: Stream Flow Control Window.  The stream level flow control
-  byte offset advertised by the sender of this parameter.
+: The maximum number of concurrent streams parameter is encoded as an unsigned
+  32-bit integer.
 
-* CFCW: Connection Flow Control Window.  The connection level flow
-  control byte offset advertised by the sender of this parameter.
+ICSL (0x0003):
 
-* MSPC: Maximum number of incoming streams per connection.
+: The idle timeout is a value in seconds that is encoded as an unsigned 16-bit
+  integer.  The maximum value is 600 seconds (10 minutes).
 
-* ICSL: Idle timeout in seconds.  The maximum value is 600 seconds (10 minutes).
+An endpoint MAY use the following transport parameters:
 
-#### Optional Transport Parameters
+TCID (0x0004):
 
-* TCID: Indicates support for truncated Connection IDs.  If sent by a peer,
-  indicates that connection IDs sent to the peer should be truncated to 0 bytes.
-  This is expected to commonly be used by an endpoint where the 5-tuple is
-  sufficient to identify a connection.  For instance, if the 5-tuple is unique
-  at the client, the client MAY send a TCID parameter to the server.  When a
-  TCID parameter is received, an endpoint MAY choose to not send the connection
-  ID on subsequent packets.
+: The truncated connection identifier parameter indicates that packets sent to
+  the peer can omit the connection ID.  This can be used by an endpoint where
```

Comment:
"sent to the peer" --> "sent to the endpoint" (since line 781 says "An endpoint MAY use the following transport parameters"